openTSNE/tsne.py

```diff
@@ -984,7 +986,7 @@ def __init__(
         min_grad_norm=1e-8,
         max_grad_norm=None,
         n_jobs=1,
-        neighbors="approx",
+        neighbors="auto",
```

Comment:
`"auto"` is nice, but then it might be a good idea to have it switch between exact search and approximate search based on the number of neighbors. Otherwise it's more of _auto_for_approx_search_.

Reply:
As I wrote in some other discussion, the speed increase using exact search is well below 1s, so I thought it's not worth the effort. But I could make it default to `BallTree` for N<1000 (?) if you prefer.

Reply:
Oh, I see, I must have missed that. In that case, it's safe to ignore it.